src/Auth/Source/External.php

```diff
@@ -225,13 +230,13 @@ private function getUser() {
 
     if (!empty($drupaluid)) {
 
-      $current_dir = getcwd();
+      $a = getcwd();
       chdir(DRUPAL_ROOT);
 
       // load the user object from Drupal
-      $drupaluser = user_load($drupaluid, TRUE);
+      $drupaluser = user_load($drupaluid);
 
-      chdir($current_dir);
+      chdir($a);
```

Comment:
Those changes are not related to SSP 2.0.x support. Please revert them

Reply:
I reset the changes.